app/models/user.rb

```diff
@@ -14,6 +15,9 @@ class User < ActiveRecord::Base
             length: { minimum: 3 }
   validates :password_confirmation, presence: true
   validates :email, uniqueness: true, presence: true
+  validates :locale, presence: true,
+            inclusion: { in: I18n.available_locales.map(&:to_s),
+                         message: 'Выберите локаль из выпадающего списка.' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.